tests/test_project/test_app/postgres_views.py

```diff
@@ -96,5 +106,29 @@ class SimpleMaterializedView(PostgresMaterialisedViewMixin, PostgresViewFromQuer
     prefix = 'test'
     pk_field = 'id'
 
-    def get_queryset(self):
+    @classmethod
+    def get_queryset(cls):
         return TestModel.objects.values()
+
+
+class ReadableTestViewFromQueryset(ReadableViewFromQueryset):
```

Comment:
Could we add a separator like we did in the others here?

```
# -----------------------------------------------------------------------------
# Readable Views
# -----------------------------------------------------------------------------
```

Reply:
Done